.changeset/free-poets-run.md

```diff
@@ -0,0 +1,5 @@
+---
+"sit-onyx": minor
+---
+
+fix(OnyxDataGrid-Filtering): removed filting config from storybook example
```

Comment:
The change only affects the docs so no changeset is needed here because there is nothing to release :)